nessie/src/test/java/org/apache/iceberg/nessie/TestNessieTable.java

```diff
@@ -579,6 +584,44 @@ public void testGCEnabled() {
             "Cannot expire snapshots: GC is disabled (deleting files may corrupt other tables)");
   }
 
+  @Test
+  public void testTableMetadataFilesCleanupDisable() {
+    Table icebergTable = catalog.loadTable(TABLE_IDENTIFIER);
+
+    Assertions.assertThat(
+            icebergTable.properties().get(TableProperties.METADATA_DELETE_AFTER_COMMIT_ENABLED))
+        .isNotNull()
+        .isEqualTo("false");
+
+    icebergTable
+        .updateProperties()
+        .set(TableProperties.METADATA_DELETE_AFTER_COMMIT_ENABLED, "true")
+        .commit();
+    icebergTable
+        .updateProperties()
+        .set(TableProperties.METADATA_PREVIOUS_VERSIONS_MAX, "1")
+        .commit();
+
+    String metadataFileLocation =
+        ((BaseTable) icebergTable).operations().current().metadataFileLocation();
+    Path metadataFileLocationPath = Paths.get(metadataFileLocation.replaceFirst("file:", ""));
```

Comment:
Would it be more robust to parse `metadataFileLocation` as a URI, assert schema value, and call `.getPath()`?

Reply:
Can you paste the expected Assertions here? I think I didn't fully get it. 

Also, other places uses this metadata location string directly 
https://github.com/apache/iceberg/blob/39a5f503c1d19d41a15b74ea53e3d5e51d495874/hive-metastore/src/test/java/org/apache/iceberg/hive/HiveTableTest.java#L238

Reply:
If it's a common pattern in test code, I'm fine with following it in Nessie tests.